### PR TITLE
PDA-149 dashboard Overtime graph style improvements

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/OvertimeGraph/BarAreaChart.js [hunk 1]
```diff
@@ -1,26 +1,81 @@
...
+      verticalAlign: 'middle',
+    },
+  },
+};
+
+const BarAreaChart = ({ data, legends, classes }) => {
+  const CustomLegend = ({ payload }) => (
+    <ul className={classes.legend}>
+      {payload.map(entry => (
+        <li key={JSON.stringify(entry)}>
```
Comment (rimselis) [inline widgets/src/components/OvertimeGraph/BarAreaChart.js]: `entry.value` not unique?

File: widgets/src/components/OvertimeGraph/BarAreaChart.js [hunk 2]
```diff
@@ -1,26 +1,81 @@
...
+          height={1}
+          wrapperStyle={{ left: 105, top: 20 }}
+          content={CustomLegend}
+        />
+      </ComposedChart>
+    </ResponsiveContainer>
+  );
+};
+
+// formatter={value => <span style={{ verticalAlign: 'middle' }}>{legends[value]}</span>}
```
Comment (rimselis) [inline widgets/src/components/OvertimeGraph/BarAreaChart.js]: needed?

File: widgets/src/components/OvertimeGraph/BarAreaChart.js (hunk 1 above)
Comment (jeffgo10) [reply widgets/src/components/OvertimeGraph/BarAreaChart.js]: can be. updated to entry.value

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/OvertimeGraph/BarAreaChart.js (hunk 2 above)
Comment (jeffgo10) [reply widgets/src/components/OvertimeGraph/BarAreaChart.js]: thank you for spotting that 👍

**Review by raxkaynan — APPROVED** [review]

**Pull request merged**